app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedFragment.java

```diff
@@ -0,0 +1,153 @@
+package com.iitp.njack.iitp_connect.core.facebook.feed;
+
+import android.arch.lifecycle.ViewModelProvider;
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.iitp.njack.iitp_connect.R;
+import com.iitp.njack.iitp_connect.common.di.Injectable;
+import com.iitp.njack.iitp_connect.data.facebook.FacebookPost;
+import com.iitp.njack.iitp_connect.data.network.Status;
+import com.iitp.njack.iitp_connect.databinding.FragmentFacebookFeedBinding;
+import com.iitp.njack.iitp_connect.ui.ViewUtils;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import static com.iitp.njack.iitp_connect.ui.ViewUtils.showView;
+
+
```

Comment:
Remove unnecessary extra line.

Reply:
That's okay. There are too many formatting issues in the project. I'll sort them out all at once. I'll merge this one.